src/grafana_service.ts

```diff
@@ -26,6 +26,75 @@ const CHUNK_SIZE = 50000;
  * @param metric to query to Grafana
  * @returns { values: [time, value][], columns: string[] }
  */
+
+export class DatasourceRequest {
+
+  public metric: Metric = undefined;
```

Comment:
please don't do it. Never expose fields like this. Use https://stackoverflow.com/questions/12827266/get-and-set-in-typescript